src/routes/userRoute/userRoutes.js

```diff
@@ -3,11 +3,11 @@ import userController from "../../controllers/user/user";
 import Authentication from "../../middlewares/authenticate";
 
 const router = Router();
-const { verifyToken } = Authentication;
+const { verifyToken, verifyUserById } = Authentication;
```

Comment:
change everything back to what was there before. I suppose that is `verifyAdmin`